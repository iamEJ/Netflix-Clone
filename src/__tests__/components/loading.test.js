import React from "react";
import { render } from "@testing-library/react";
import Loading from "../../components/loading";

describe("<Loading />", () => {
  it("renders <Loading /> ", () => {
    const { container, getByTestId } = render(
      <Loading src="/images/spinner.png" data-testid="loading" />
    );
    expect(getByTestId("loading")).toBeTruthy();
    expect(getByTestId("loading-picture")).toBeTruthy();
    expect(container.firstChild).toMatchSnapshot();
  });

  it("renders <Loading.ReleaseBody /> ", () => {
    const { container } = render(<Loading.ReleaseBody />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
