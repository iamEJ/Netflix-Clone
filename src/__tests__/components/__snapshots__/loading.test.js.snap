// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loading /> renders <Loading />  1`] = `
<div
  class="loading__Spinner-sc-1f7p2o9-0 jUTHDM"
  data-testid="loading"
>
  <img
    class="loading__Picture-sc-1f7p2o9-1 dUMkVl"
    data-testid="loading-picture"
    src="https://mir-s3-cdn-cf.behance.net/project_modules/disp/64623a33850498.56ba69ac2a6f7.png"
  />
</div>
`;

exports[`<Loading /> renders <Loading.ReleaseBody />  1`] = `null`;
